src/other/utils.jl

```diff
@@ -75,3 +75,7 @@ function funname(f)
     n = nameof(f)
     String(n)[1] == '#' ? :function : n
 end
+
+if isdefined(Base, :ComposedFunction)
```

Comment:
Maybe make this a `VERSION` check? That will be easier to clean when we drop support for older versions.

Also add a test?

Reply:
OK

Reply:
do you remember how to get the exact version number?

Reply:
With contrib/commit_name.sh. This one is `1.6.0-DEV.85`.